### PR TITLE
can now set MYSQL_PASSWORD in env

### PR DESCRIPTION
**Pull request by jedp**
made a note in the TESTING.md doc

example:

```
NODE_ENV=test_mysql MYSQL_USER=browserid MYSQL_PASSWORD=browserid npm 
```

Issue #1560

### CODE REVIEW

File: docs/TESTING.md [hunk 1]
```diff
@@ -4,6 +4,13 @@
...
-  # `npm test`
+  - `npm test`
 
-  # Load http://localhost:10002/test/index.html into a world wide web browser
+  - Load http://localhost:10002/test/index.html into a world wide web browser
+
+Note that for mysql, you will need to grant `browserid` privileges to create tables.
+You can then run the mysql suite with, e.g., 
+
+    NODE_ENV=test_mysql MYSQL_USER=browserid MYSQL_PASSWORD=browserid npm test
```
Comment (ozten) [inline docs/TESTING.md:14]: Thanks for documenting!

File: lib/configuration.js [hunk 2]
```diff
@@ -93,7 +93,10 @@ var conf = module.exports = convict({
       format: 'string?',
       env: 'DATABASE_NAME'
     },
-    password: 'string?',
+    password: {
+      format: 'string?',
+      env: 'MYSQL_PASSWORD'
```
Comment (ozten) [inline lib/configuration.js:98]: Not having this has been an pain, thanks for fixing this!

**Comment by ozten** [comment]
Tests pass http://travis-ci.org/#!/mozilla/browserid/builds/1281295
r+

**Comment by jrgm** [comment]
verified in stage train-2012.05.14

**Pull request merged**